CHERI-UPDATING.md

```diff
@@ -79,6 +79,152 @@ Each merge from upstream FreeBSD to [dev] is accompanied by a tag of the form
 We typically merge to [master] from [dev] at stable points at least a week
 apart.
 
+### The merge process
+CheriBSD is extremely diverged from FreeBSD with nearly two
+thousand changed files. As such, there are many opportunities for
+a merge to conflict and worse for a merge to not conflict, but
+be broken. It is critical that we be able to bisect the merged
+changes. To support this we merge changes one commit at a time
+from the _first-parent_ history of FreeBSD's `main` using the
+[mergify](https://github.com/brooksdavis/mergify) script. This makes it
+straightforward to discover which change caused a problem.
+
+This section describes the merge process in detail.
+
+#### Setup
+The merger must add an appropriate FreeBSD remote to their setup. This
+example uses the official FreeBSD mirror, a mirror on GitHub or the like
+will also work fine.
+```
+$ git remote add upstream https://git.FreeBSD.org/src.git
+```
+You will also want to enable `git revere` to simplify handling the same
+conflicting merges over again. To do so globally, run:
+```
+$ git config --global rerere.enabled true
+```
+Disable rename detection. This sometimes leads to awkward merges but
+rename detection takes **forever** on the CheriBSD repo.
```

Comment:
Want to give the command?

Reply:
My guess is `git config merge.renames false`

FWIW, I did this as a local config in the checkout vs doing it globally.